### DISCUSSION TITLE
Table of Contents

### DISCUSSION THREAD

**Discussion by TelegramSam** [decentralized-identity/didcomm-book#2] [Ideas]
I want to gather ideas on what the DIDComm book Table of Contents would look like. Don't worry about what's there now, just what you want to be there. Also, no worries to have a fully complete answer.

**Comment by TelegramSam** [comment]
From Tim Spring:

Table of contents:

What is DIDComm
Key Terms
The Basics
	Verifiable Credentials? Make more clear 
	Cryptographic security - default / how it works
More technically how it works

	Why DIDComm?
		What are you building? Decision making process
Highlight pain point not currently being addressed
Compelling reason to put effort into switching to DIDComm
	Main message - Communicate securely between people / apps

	Real world use cases / Examples - different applications 
		Real world example 1
		Real world example 2

	DIDComm vs Alternative Solutions
		You might think this sounds similar to X
			How DIDComm compares / is different
				(Maybe reach out to other community - get their thoughts, direct to our resource)
		You might think this sounds similar to Y
			How DIDComm compares / is different
				

	Safety / Privacy Concerns

	Technical how to
		Get started
		Troubleshooting

	Questions? / Learn more
		Join the working group

**Comment by FabioPinheiro** [comment]
In `What is DIDComm`
I would go further and say what is he trying to achieve.
Because probably all the scenarios that you can imagine can also be implemented with traditional/proven technologies.
Imagine that you want to change credentials why would you use DID Comm instead of using the signal protocol (or any other)? What does it offer on top of it?

I think it will be important to make some of those points more clear:
- Is DID Comm a message format?
- Is DID Comm a channel for communication?
- Is DID Comm a communication methodology as mentioned on the specs? what does that even mean?
- How would the future be? For example, now it seems that DID Comm depends on DNS for a lot of functionalities. If I want to do something new why should I invest in DID Comm instead of using just traditional technologies?
